lib/decorators/asyncapi-operation.decorator.ts

```diff
@@ -38,8 +38,9 @@ function makeMessage(
   };
 }
 
-export function AsyncApiOperation(
-  ...options: AsyncApiOperationOptions[]
+export function AsyncApiOperationForMetaKey(
```

Comment:
let's move that decorator to a separate file

Reply:
lib/decorators/asyncapi-operation-for...

Reply:
Hey guys. Need this feature. I've moved function to separate file, modify sample and tests for it:
https://github.com/flamewow/nestjs-asyncapi/pull/473